main/reference/zoe-api/zoe-helpers.md

```diff
@@ -71,61 +74,63 @@ of the *transfer* argument of the **[atomicRearrange()](#atomicrearrange-zcf-tra
 - Returns: None.
 
 Asks Zoe to rearrange the **[Allocations](./zoe-data-types#allocation)** among the seats mentioned in
-*fromSeat* and *toSeat*. The reallocations must satisfy several constraints. If these constraints are all
+_fromSeat_ and _toSeat_. The reallocations must satisfy several constraints. If these constraints are all
 met, then the reallocation happens atomically. Otherwise an error is thrown and none of the proposed
 changes has any effect. The constraints are as follows.
 
- * All the mentioned seats are still live.
- * There aren't any outstanding stagings for any of the mentioned seats.
+- All the mentioned seats are still live.
+- There aren't any outstanding stagings for any of the mentioned seats.
 
-	Stagings are a reallocation mechanism that has been
-    deprecated in favor of this **atomicRearrange()** function.
-    To prevent confusion, each reallocation can only be
-    expressed in the old way or the new way, but not a mixture.
- * Overall conservation must be maintained. In other words, the reallocated
-    **[Amounts](/reference/ertp-api/ertp-data-types#amount)** must balance out.
- * Offer Safety is preserved for each seat. That means reallocations can only take assets from a seat
-	 as long as either it gets the assets described in the want section of its proposal, or it retains
-     all of the assets specified in the give section of the proposal. This constraint applies to each
-     seat across the entire atomicRearrangement, not to the individual **TransferParts**.
+      	Stagings are a reallocation mechanism that has been
 
```

Comment:
This is still broken.  Compare:
![stagings-wrong](https://github.com/Agoric/documentation/assets/13372636/75fca2b7-837d-498b-8f32-f5f494f70404)

== with ======

![Stagings-orig](https://github.com/Agoric/documentation/assets/13372636/cfdb3157-f0ac-46e5-a352-b12b0d3cabc6)